cmd/stdiscosrv/apisrv.go

```diff
@@ -359,16 +370,16 @@ func certificateBytes(req *http.Request) []byte {
 	}
```

Comment:
Could add an "else" here to return a more specific error, e.g. `no certificate in request`.

Reply:
`else` may not required here. Already have `if bs == nil` followed

Reply:
My intention was that there was a header, but it is e.g. empty, thus `bs == nil`. The else would then differentiate that from not having any header at all. Deep bikeshedding territory :D